core/https-client-daemon/src/lib.rs

```diff
@@ -34,13 +34,17 @@ pub mod sgx_reexport_prelude {
 use itp_types::AccountId;
 pub mod daemon_sender;
 pub mod error;
-pub mod https_client;
-
+// pub mod https_client;
 pub use error::Result;
 
-use std::string::String;
+use codec::{Decode, Encode, MaxEncodedLen};
+use litentry_primitives::{Identity, Web2ValidationData};
 
+#[derive(Encode, Decode, Clone, Debug, PartialEq, Eq, MaxEncodedLen)]
 pub struct Request {
```

Comment:
Maybe call this `TwitterIdentityVerificationRequest` or any shorter name 😂  we'll have more requests that have different fields.

Reply:
Actually, I refactored the https further. 😂  I think this refactoring is what you think. 
See at:  https://github.com/litentry/tee-worker/commit/2d8c835c0d4ecb7386faeb8b7256413a2e308266#diff-131f0a349f74975ed71970e0a98d237e597f0130008bf5cf0c5c6e6943d81dad.

Maybe we can do this on the next PR. What do you think?

Reply:
Let's do it in another PR. Btw there is a typo in the above link: `Web3IndentityVerification`